plugins/newsdownloader.koplugin/feeds.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ad. 3. XML is open "protocol" commonly used in many technologies, many people (even not developers) are familiar with it. 
And lua files are well known only to lua programmers. 
So much more people are ale to edit XML than lua files. 

Ad. 5. Feeds.xml contains data that MUST/should be modified by user. 

NewsConfig.lua contains data that should NOT be modified ( at least not by unexperienced users).